prac_07/convert_miles_km.kv

```diff
@@ -0,0 +1,25 @@
+BoxLayout:
+    orientation: 'vertical'
+    BoxLayout:
+        orientation: 'horizontal'
+        TextInput :
+            id : input_number
+            text: '33'
+            on_text: app.calculation()
+        BoxLayout:
+            orientation: 'vertical'
+            Button:
+                text: 'Up'
+                on_press: app.handle_increment(1)
+            Button:
+                text: 'Down'
+                on_press: app.handle_increment(-1)
```

Comment:
Could all these functions be _passed_ a value from another place in the program?